<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>streamWindow</class>
 <widget class="QMainWindow" name="streamWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>843</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Streaming</string>
  </property>
  <property name="styleSheet">
   <string notr="true">backgorund-color:#f2f8ff;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="streaming">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>431</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8px;
background-color: white;
color: black;
border: 2px solid #86bcfa;
box-shadow: rgba(0, 0, 0, 0.25) 0px 54px 55px, rgba(0, 0, 0, 0.12) 0px -12px 30px, rgba(0, 0, 0, 0.12) 0px 4px 6px, rgba(0, 0, 0, 0.17) 0px 12px 13px, rgba(0, 0, 0, 0.09) 0px -3px 5px;
padding: 5px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="plateImg">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>470</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8px;
background-color: white;
color: black;
border: 2px solid #86bcfa;
padding:3px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="endpoint">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>570</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8px;
background-color: white;
color: black;
border: 2px solid #86bcfa;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="greenImg">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>431</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8px;
background-color: white;
color: black;
border: 2px solid #86bcfa;
padding: 5px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LP">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>580</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8px;
background-color: white;
color: black;
border: 2px solid #86bcfa;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="PushStream">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8px;
font-size: 20px;
background-color: #3093ed;
color: black;
border: 2px solid #3093ed;
box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2), 0 6px 20px 0 rgba(0,0,0,0.19);
</string>
    </property>
    <property name="text">
     <string>Stream</string>
    </property>
   </widget>
   <widget class="QPushButton" name="capture">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>670</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8px;
font-size: 20px;
background-color: #3093ed;
color: black;
border: 2px solid #3093ed;
box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2), 0 6px 20px 0 rgba(0,0,0,0.19);
</string>
    </property>
    <property name="text">
     <string>Capture</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1013</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
